sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/PluginReferential.java

```diff
@@ -102,6 +102,14 @@ public List<String> findLastReleasesWithDependencies(String pluginKey) {
   }
 
   public void addOutgoingDependency(Release release, String requiredPluginReleaseKey, String requiredMinimumReleaseVersion) {
+
+    // skip dependencies on license, as it's deprecated and is provided by SQ anyway. There is no need
+    // to have the explicit dependency on the update center side.
+    // Once we removed plugins < 8.9, and bundled plugins are removed from the matrix, we can drop this if
```

Comment:
I think we are missing the end of the comment?

Reply:
I updated the message to clarify 👍